content/en/docs/components/pipelines/introduction.md

```diff
@@ -1,284 +1,60 @@
-
-## Architectural overview
-
-<img src="/docs/images/pipelines-architecture.png" 
-  alt="Pipelines architectural diagram"
-  class="mt-3 mb-3 p-3 border border-info rounded">
-
-At a high level, the execution of a pipeline proceeds as follows: 
+A [_pipeline_][pipelines] is a description of an multi-step workflow, where each step is defined by a single container execution. Each step, or [_task_][tasks], is parameterized by inputs and outputs, enabling pipeline authors to form a computational directed acyclic graph (DAG) of tasks by specifying the output of one task as the input to another.
 
-* **Python SDK**: You create components or specify a pipeline using the Kubeflow
-  Pipelines domain-specific language 
-  ([DSL](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8/sdk/python/kfp/dsl)).
-* **DSL compiler**: The
-  [DSL compiler](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8/sdk/python/kfp/compiler)
-  transforms your pipeline's Python code into a static configuration (YAML).
-* **Pipeline Service**: You call the Pipeline Service to create a
-  pipeline run from the static configuration. 
-* **Kubernetes resources**: The Pipeline Service calls the Kubernetes API server to create
-  the necessary Kubernetes resources
-  ([CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/))
-  to run the pipeline.
-* **Orchestration controllers**: A set of orchestration controllers
-  execute the containers needed to complete the pipeline.
-  The containers execute within Kubernetes Pods on virtual machines.
-  An example controller is the
-  **[Argo Workflow](https://github.com/argoproj/argo-workflows)** controller,
-  which orchestrates task-driven workflows.
-* **Artifact storage**: The Pods store two kinds of data: 
+Pipelines are written in Python to enable an easy authoring experience, compiled to YAML for portability, and executed on Kubernetes for scalability.
 
-  * **Metadata:** Experiments, jobs, pipeline runs, and single scalar metrics.
-    Metric data is aggregated for the purpose of sorting and filtering.
-    Kubeflow Pipelines stores the metadata in a MySQL database.
-  * **Artifacts:** Pipeline packages, views, and large-scale metrics (time series).
-    Use large-scale metrics to debug a pipeline run or investigate an individual run’s performance.
-    Kubeflow Pipelines stores the artifacts in an artifact store like
-    [Minio server](https://docs.minio.io/) or
-    [Cloud Storage](https://cloud.google.com/storage/docs/).
 
-    The MySQL database and the Minio server are both backed by the Kubernetes
-    [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes)
-    subsystem.
+## What does using KFP look like?
+At a high level, a typical KFP user experience is as follows:
+1. [Author a pipeline][author-a-pipeline] using the **Python KFP SDK**'s domain-specific language (DSL)
```

Comment:
A first step might be authoring a component, or reuse an prebuilt component. Not sure if we need to mention this though. Up to you.